cdr/enterprise-images
# Base images

### What This Does

* Creates a set of base images at `codercom/enterprise-base` that other images use
  * Base images come in `ubuntu`, `centos`, and `arch` flavors -- `rhel` will come in a later PR
  * Packages match the spec outlined in [OOTB RFC](https://github.com/cdr/c/blob/master/rfcs/0045-ootb-images.md#phase-2---images)
  * Some images now have fewer packages than before, hopefully this doesn't disrupt anything but I wanted to keep it as slim as the spec outlines
* Creates a set of multi-editor compatible base images at `codercom/enterprise-multieditor`
  * All of the jetbrains IDE images use these
* Adds an enforced order to the ci generator so that images are built in order of base -> multieditor -> all others

### Steps to Test

1. Make sure CI passes
2. Pull some of the images, make sure they have all of the dependencies as outlined in RFC 0045
3. Import some of the images into Coder Enterprise, confirm they work as advertised
